Jenkinsfile

```diff
@@ -10,18 +8,16 @@ elifePipeline {
 
         stage 'Build images', {
             checkout scm
-            dockerComposeBuild(commit)
-        }
-
-        stage 'Check version', {
-            candidateVersion = dockerComposeRunAndCaptureOutput(
-                "sciencebeam",
-                "./print_version.sh",
-                commit
-            ).trim()
-            echo "Candidate version: ${candidateVersion}"
-            isNew = sh(script: "git tag | grep v${candidateVersion}", returnStatus: true) != 0
-            echo "isNew: ${isNew}"
+            // TODO: extract into dockerComposeBuild
+            def version
+            if (env.TAG_NAME) {
+                version = env.TAG_NAME - 'v'
+            } else {
+                version = 'develop'
```

Comment:
I wonder whether it should instead be blank? It's not a semantic version and pypi wouldn't accept that.

Reply:
Good for me, the Docker image label won't be very clear but such an image should never be pushed.